// echo.proto
syntax = "proto3";

option cc_generic_services = true;

message EchoRequest {
  string message = 1;
}

message EchoResponse {
  string message = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
  rpc Echo1(EchoRequest) returns (EchoResponse);
}
